packages/react-devtools-extensions/src/background/dynamicallyInjectContentScripts.js

```diff
@@ -20,7 +20,7 @@ const contentScriptsToInject = IS_FIREFOX
         js: ['build/proxy.js'],
         matches: ['<all_urls>'],
         persistAcrossSessions: true,
-        runAt: 'document_end',
+        runAt: 'document_idle',
```

Comment:
just a typo fix, accidentally missed it, should not affect in any way

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/extensionTypes/RunAt